docs/Installation/InstallationFromSource.rst

```diff
@@ -7,7 +7,7 @@ If you want to contribute to PYMEs development, or you want to use PYME in a con
 requires modification to some of the scripts you will want to have the PYME sources somewhere you can easily find, modify
 and update them.
 
-If you just want to use the data analysis features then you are strongly advised to follow the instructions in :ref:`installationanaconda`_
```

Comment:
Why delete the underscore? I'm pretty sure it's need to turn this into a link.

Reply:
For external links you need the underscore, for internal ones you need to remove it. Verified by compiling the docs.